src/documentation/sdks/python.md

```diff
@@ -3,8 +3,17 @@ layout: default
 title: "Beam Python SDK"
 permalink: /documentation/sdks/python/
 ---
-# Apache Beam Python SDK _[Under Development]_
+# Apache Beam Python SDK
 
-The Beam Python SDK is currently under development on a feature branch. Would you like to contribute? See the Beam [Work in Progress]({{ site.baseurl }}/contribute/work-in-progress/#feature-branches) page to find out how you can help!
+The Python SDK for Apache Beam provides a simple, powerful API for building both batch and streaming parallel data processing pipelines in Python.
+
+## Get Started with the Python SDK
+
+Get started with the [Beam Programming Guide]({{ site.baseurl }}/learn/programming-guide) to learn the basic concepts that apply to all SDKs in Beam.
+
+Then, follow the [Beam Python SDK Quickstart]({{ site.baseurl }}/get-started/quickstart-py) to set up your Python development environment, get the Beam SDK for Python, and run an example pipeline.
+
+## Python Type Safety
+
+Python is a dynamically-typed language with no static type checking. The Beam SDK for Python uses type hints during pipeline construction and runtime to try to emulate the correctness guarantees achieved by true static typing. [Ensuring Python Type Safety]({{ site.baseurl }}/documentation/sdks/python-type-safety) walks through how to use type hints, which help you to catch potential bugs up front in the local runner, rather than after minutes of execution into a deep, complex pipeline.
```

Comment:
Do we need the final part of this paragraph: 'rather than after minutes of execution into a deep, complex pipeline.' ? I am not opposed it, I am want to understand how much value it adds.